src/pages/components/notification/usage.mdx

```diff
@@ -4,79 +4,166 @@ description: Notifications are messages that communicate information to the user
 
 <Row>
 <Column colLg={8}>
 
-![High vs. low contrast](images/notification-usage-4.png)
+![Example of inline notification placement](images/notification-placement-inline.png)
 
 </Column>
 </Row>
 
-## Placement
-
 #### Toast notifications
 
-Toast notifications slide in and out a page from the top-right corner. Actionable notifications do not appear on mobile screen widths.
+Toast notifications slide in and out from the top right of the screen. They align to the edge of the last column and can stack with `$spacing-03` in-between. New toast notifications should appear at the top of the list, with older notifications being pushed down until they are dismissed.
+Actionable toast notifications do not appear on mobile screen widths.
 
 <Row>
 <Column colLg={8}>
 
-![Toast notification example](images/notification-usage-2.png)
+![Example of toast notification placement](images/notification-placement-toast.png)
 
 </Column>
 </Row>
 
-#### Inline notifications
+## Behavior
+
+### Dismissal
+
+We recommend that toast notifications automatically disappear after five seconds. Inline notifications are persistent until the user dismisses them. All notifications have at least one method of dismissal which is typically a small “x” in the upper right hand corner.
+
+### Overflow
+
+Notification messages should be kept short. For toast notifications, the timestamp is optional and can be removed if a third line of content is needed.
+
+If a toast or inline notification requires a message of longer than three lines, include a short message with a “View more” link that takes the user to a view of the full notification message. This can be either a full page with more details or a modal.
+
+## Content
+
+Notifications provide limited space for content, and therefore the content must be concise. A user should be able to quickly scan the notification, be apprised of the situation, and know what to do next.
+
+### Title
+
+- The title should be short and descriptive, explaining the most important piece of information.
+- When possible, communicate the main message using just the title.
+- Use a period only if the title is a full sentence.
 
-Inline notifications appear near its related item. In [forms](/components/form/code), we recommend placing the inline notification at the bottom of the form, right before the submission buttons. Depending on the context of the page, inline notifications can appear above the content as well.
+### Body content
+
+- The body of the notification should be no more than two lines.
+- Be concise and avoid repeating or paraphrasing the title.
+- Explain how to resolve the issue by including any troubleshooting actions or next steps. You can include links within the notification body that redirect the user to next steps.
```

Comment:
How are we anticipating keyboard or screen reader users to navigate to the toast if there is interactive content inside? By default, it seems like these might be hard to reach quickly within a small time window.

Reply:
I think this is more talking to inline notification that toast. But i think it would be good to emphasis that point here. Like 

> "If you include links within the notification body that redirect the user to steps, be sure keyboard and screenreader user have enough time to access the notification or can easily find the notification after it disappears."